main.rb

```diff
@@ -9,17 +9,14 @@
 interlude_bars = 2
 
 synthia_type = :dpulse
-synthia_amp = 0.5
+synthia_amp = 0.1
```

Comment:
```suggestion
synthia_amp = 0.3
```
0.1 ist etwas zu leise finde ich, ich höre es kaum noch :D